Documentation/using-rkt-with-systemd.md

```diff
@@ -10,6 +10,47 @@ This document describes how to use `rkt` with `systemd`.
 Since `rkt` consists only of a simple CLI that directly executes processes and writes to stdout/stderr (i.e. it does not daemonize), the lifecycle of `rkt` pods can be directly managed by `systemd`.
 Consequently, standard `systemd` idioms like `systemctl start` and `systemctl stop` work out of the box.
 
+## systemd-run
+
+If you just want to start a daemonized container you don't need to create a systemd unit file, you can just use `systemd-run`:
```

Comment:
perhaps "to start a daemonized container from the command line"; also, could you please linkify systemd-run? http://www.freedesktop.org/software/systemd/man/systemd-run.html